benchmarks/dbscan/ArborX_DBSCANVerification.hpp

```diff
@@ -254,12 +254,12 @@ bool verifyClustersAreUnique(ExecutionSpace const &exec_space,
   }
   if (cluster_sets.size() != num_unique_cluster_indices)
   {
-    std::cerr << "Number of components does not match" << std::endl;
+    std::cerr << "Number of components does not match" << '\n';
```

Comment:
I'd expect we intended to flush here but if we didn't then the `\n` belongs to arrays and should not be appended as a single char.

Reply:
`std::cerr` isn't buffered so we don't need to flush.

Reply:
I did not know that :)